docs/tutorial/desktop-environment-integration.md

```diff
@@ -8,56 +8,6 @@ applications can put a custom menu in the dock menu.
 This guide explains how to integrate your application into those desktop
 environments with Electron APIs.
 
-## Notifications (Windows, Linux, macOS)
```

Comment:
Maybe we should leave this H2 in place and link to the new doc, in case anyone is hyperlinking to it.